starter_code/src/movies.js

```diff
@@ -2,8 +2,17 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 const orderByYear = (array) => {
-    const sorted = obj["year"].sort( (a,b)=> a-b).join('')
-    const newArray = array.map(obj =>  newArray)
+    const newArray = [...array] //guarda array nueva
+    const sorted = newArray.sort((a,b)=> {
+        if (a.year === b.year){
+            // newArray.sort((a,b =>
+        } else {
+
+        }
+        a.year-b.year
+    })
+
+    return sorted
```

Comment:
Les recomiendo revisen la documentacion de **sort()**
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort](url)
<img width="601" alt="Screenshot 2020-01-13 at 10 46 55" src="https://user-images.githubusercontent.com/35221964/72274575-104c1200-35f2-11ea-9739-571f2b8f3c8f.png">

> To compare numbers instead of strings, the compare function can simply subtract b from a. The following function will sort the array in ascending order (if it doesn't contain Infinity and NaN):